solr/core/src/java/org/apache/solr/handler/admin/CoreAdminOperation.java

```diff
@@ -172,12 +175,14 @@ public enum CoreAdminOperation implements CoreAdminOp {
       RENAME,
       it -> {
         SolrParams params = it.req.getParams();
-        String name = params.required().get(CoreAdminParams.OTHER);
-        String cname = params.required().get(CoreAdminParams.CORE);
```

Comment:
As we are moving our code from CoreAdminOperation to API. I think it would be great to remove this validation from here as we are already testing required parameter using `ensureRequiredParameterProvided` method in APIs.